src/pkgcheck/checks/codingstyle.py

```diff
@@ -79,6 +79,29 @@ def feed(self, pkg):
                     yield DeprecatedEapiCommand(name, line=call, lineno=lineno+1, eapi=pkg.eapi, pkg=pkg)
 
 
+class EendMissingArg(results.LineResult, results.Style):
+    """Eend is called without an argument"""
+
+    @property
+    def desc(self):
+        return f'Call to eend is missing an argument, on line {self.lineno}'
+
+
+class EendMissingArgCheck(Check):
```

Comment:
I'd combine this check with the previous ebegin/eend related check.

Reply:
I'd rather not do that at this moment. This check is simple, because it can just focus on one line, the ebegin-eend matching requires keeping some context around.

If they're combined, I'd rather have this added first, and then return with an implementation for ebegin-eend.